cluster:
  ros__parameters:
    publisher_cluster_url: "perception/lidar/cluster"
    publisher_cluster_merged_points_url: "perception/lidar/cluster_merged_points"
    subscription_url: "perception/lidar/ground_filter_outliers"
    cluster_tolerance: 0.5
    min_cluster_size: 20
    max_cluster_size: 200